dotCMS/src/main/java/com/dotcms/rest/api/v1/page/ContentletMixIn.java

```diff
@@ -0,0 +1,12 @@
+package com.dotcms.rest.api.v1.page;
+
+import com.dotmarketing.business.Permissionable;
+import com.dotmarketing.portlets.structure.model.Structure;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+public abstract class ContentletMixIn {
+
+    @JsonIgnore abstract Permissionable getParentPermissionable();
+    @JsonIgnore abstract public Structure getStructure() ;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)